tests/Aspire.Hosting.AWS.Tests/AWSSchemaTests.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Amazon;
+using Aspire.Hosting.Tests.Schema;
+using Xunit;
+
+namespace Aspire.Hosting.AWS.Tests;
+
+public class AWSSchemaTests
+{
+    [Fact]
+    public void ValidateAwsStackSamples()
```

Comment:
I know other tests share the name, but the name is kinda misleading. `ValidateApplicationSamples` is only validating the generated manifest, and there is no "sample" project as such. Would it be better to name the tests after `AddAWSCloudFormationStack`/`AddAWSClouldFormationTemplate`?

Reply:
Thanks, was wondering what name to use.